scripts/get_countries_map_data.R

```diff
@@ -1,34 +1,37 @@
-map_url <- paste0(
-  "http://www.naturalearthdata.com/http//www.naturalearthdata.com/",
-  "download/50m/cultural/ne_50m_admin_0_countries.zip"
-)
+# Note: if the map_url below fails to work, you can also get the dataset
+# manually from the link below:
+# https://hub.arcgis.com/datasets/esri::world-countries-generalized/explore?location=-0.062553%2C0.000000%2C2.00
+
+map_url <-
+  "https://opendata.arcgis.com/api/v3/datasets/2b93b06dc0dc4e809d3c8db5cb96ba69_0/downloads/data?format=shp&spatialRefId=4326&where=1%3D1"
 
-file <- paste0("./scripts/raw_data/", basename(map_url))
+file <- paste0("./scripts/raw_data/", basename(map_url), ".zip")
 
 if (!file.exists(file)) download.file(map_url, file)
```

Comment:
The script seems to fail here. Let's make sure it works cleanly. It doesn't have to be fully automated - if download is an issue, a comment saying "download file from this link and put it here" is fine.

Reply:
Interestingly the download works on my end and the script works seamlessly. Can you point me out to the specific error message so I can look into why it isn't working on your end? I've added a screenshot below for the same.

<img width="773" alt="Screenshot 2023-02-16 at 1 35 44 PM" src="https://user-images.githubusercontent.com/26517718/219304797-c8a5385a-4391-4c12-9a12-6ce0d32d4056.png">

Reply:
False alarm. I didn't notice that the script expects to be ran from the parent directory.